Automattic/wp-calypso
# Site Level User Profile: revert banner on /me and add notice on /me/account

Related to:

- https://github.com/Automattic/dotcom-forge/issues/8645

## Proposed Changes

This removes this "site-level user profile" banner on `/me` introduced by https://github.com/Automattic/wp-calypso/pull/93280:

<img width="516" alt="image" src="https://github.com/user-attachments/assets/a75fd36e-ef14-40f5-a7cc-59534ebfc2b2">

<br><br>

Additionally, under the same feature flag (`layout/site-level-user-profile`), it shows a notice about admin color scheme in `/me/account`, saying that we can now set it individually on Users -> Profile:

|Before|After|
|-|-|
|<img width="1010" alt="image" src="https://github.com/user-attachments/assets/fb582995-4acd-4579-b80d-9f8f3756659b">|<img width="1004" alt="image" src="https://github.com/user-attachments/assets/02cf5be7-228c-43a8-9158-86b398ca6952">|

## Testing Instructions

1. Use the Calypso live URL (which enables the `layout/site-level-user-profile` flag by default)
2. Go to /me and verify that no banner is shown.
3. Go to /me/account and verify that the color scheme setting is now replaced by a notice.

## Pre-merge Checklist
